jujugui/static/gui/src/app/components/deployment-flow/credential/add/test-add.js

```diff
@@ -76,6 +76,22 @@ describe('DeploymentCredentialAdd', function() {
       },
       message: 'a message'
     });
+    getCloudProviderDetails.withArgs('maas').returns({
+      id: 'maas',
+      showLogo: false,
+      title: 'MAAS',
+      forms: {
+        'access-key': [{
+          id: 'access-key',
+          title: 'The EC2 access key'
```

Comment:
Even if this is just used to set up a mock, it's weird to see ec2 stuff on a maas section.

Reply:
Oops, thanks.